engines/stark/services/userinterface.cpp

```diff
@@ -134,24 +138,26 @@ void UserInterface::requestFMVPlayback(const Common::String &name) {
 }
 
 void UserInterface::onFMVStopped() {
-	backPreScreen();
+	backPrevScreen();
```

Comment:
This commit contains some changing of names like this.  Just my own decision. Nothing remarkable.